xml/System.Buffers/ReadOnlySequenceSegment`1.xml

```diff
@@ -47,7 +47,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Initializes a new read-only sequence segment instance.</summary>
```

Comment:
Style guide: https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Constructor

```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Buffers.ReadOnlySequenceSegment`1" /> struct.</summary>
```

Reply:
@mairaw `ReadOnlySequenceSegment` is a class.

Reply:
Fixed. I was looking at the ReadOnlySequence source code when I provided the suggestion.